### PR TITLE
Add test checking the presence of deviceId in screen share video track

### PR DESCRIPTION
**Pull request by youennf**
Test for https://github.com/w3c/mediacapture-screen-share/pull/310

### CODE REVIEW

File: screen-capture/getdisplaymedia-settings.https.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+}, "getDisplayMedia() deviceId setting and capability");
+
+promise_test(async t => {
+  const stream = await getDisplayMedia({video: true});
+  const track = stream.getVideoTracks()[0];
+  t.add_cleanup(() => stopTracks(stream));
+
+  const settings = track.getSettings();
+  const capabilities = track.getCapabilities();
+alert(JSON.stringify(capabilities));
```
Comment (alvestrand) [inline screen-capture/getdisplaymedia-settings.https.html:42]: Should you remove this before submitting? alert() is a blocking call....

**Review by alvestrand — APPROVED** [review]
Approving, with nit.

File: screen-capture/getdisplaymedia-settings.https.html (hunk 1 above)
Comment (youennf) [reply screen-capture/getdisplaymedia-settings.https.html:42]: Oops, will fix this.

Comment (youennf) [reply screen-capture/getdisplaymedia-settings.https.html]: Done

**Review by jan-ivar — APPROVED** [review]

**Pull request merged**